core/src/wallet/common/api_impl/OperationApi.h

```diff
@@ -31,6 +31,8 @@
 #ifndef LEDGER_CORE_OPERATIONAPI_H
 #define LEDGER_CORE_OPERATIONAPI_H
 
+#define CORRELATIONID_PREFIX(id) id.empty()?"":(std::string("corrId=")+id)
```

Comment:
The tag should be named `ledger.correlation_id` according to https://ledgerhq.atlassian.net/wiki/spaces/VAUL/pages/2886074741/Logging+Guidelines#Normalising-the-data